public/restaurant.js

```diff
@@ -0,0 +1,67 @@
+let path = window.location.pathname
+let pathArray = path.split('/')
+let id = pathArray.pop()
+let name = document.getElementById('name')
+let address = document.getElementById('address')
+let phone = document.getElementById('phone')
+let website = document.getElementById('webSite')
+let hours = document.getElementById('hours')
+let notes = document.getElementById('notes')
+let noteContainer = document.getElementById('noteList')
+
+// get restaurant data
+async function getRest(id) {
+    let restItem = await fetch(`https://json-server.burlingtoncodeacademy.now.sh/restaurants/${id}`)
+        .then((response) => {
+            return response.json()
+        })
+        .then((dataJson) => {
+            return dataJson
+        })
+
+    restId.textContent = restItem.id
+    name.textContent = restItem.name
+    address.textContent = restItem.address
+    phone.textContent = restItem.phone
+    webSite.textContent = restItem.website
+    hours.textContent = restItem.hours || 'Hours not available.'
```

Comment:
Using the `||` here is a great catch. It wouldn't be a terrible idea to put it on the other ones as well in case there isn't a phone number or website ect.